### PR TITLE
Added run only failed tests attribute

### PR DESCRIPTION
**Pull request by matllubos**

### CODE REVIEW

File: junorunner/testrunner.py [hunk 1]
```diff
@@ -30,18 +37,27 @@ def add_arguments(cls, parser):
...
 
     def run_tests(self, test_labels, extra_tests=None, **kwargs):
         """
         Run the unit tests for all the test labels in the provided list.
         """
 
+        if self.only_failed:
+            with open(RERUN_LOG_FILE_NAME, 'r') as f:
+                test_labels = f.read().split('\n')
+            test_labels += [':']
```
Comment (rubickcz) [inline junorunner/testrunner.py:55]: Why?

File: junorunner/testrunner.py [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
 from django.core import management
 
 from .runner import JunoDiscoverRunner
 
 
+RERUN_LOG_FILE_NAME = getattr(
+    settings,
+    'TEST_RUNNER_RERUN_LOG_FILE_NAME',
+    'test_rerun.txt'
+)
```
Comment (rubickcz) [inline junorunner/testrunner.py:12]: I don't see any writing to this file...

Comment (matllubos) [reply junorunner/testrunner.py:12]: There is no writing there is reading. Writing is in TextTestResult in extended_runner.py (the file already exists)

File: junorunner/testrunner.py (hunk 1 above)
Comment (matllubos) [reply junorunner/testrunner.py:55]: because : seperates test case names and test names. Without : it will not work unfortunatelly

**Review by zamazaljiri — APPROVED** [review]

**Review by rubickcz — APPROVED** [review]

**Comment by rubickcz** [comment]
CI failed.

**Pull request merged**